x-pack/plugins/security_solution/public/common/components/user_privileges/endpoint/use_endpoint_privileges.ts

```diff
@@ -57,7 +58,7 @@ export const useEndpointPrivileges = (): Immutable<EndpointPrivileges> => {
             licenseService,
             fleetAuthz,
             userRoles,
-            isEndpointRbacEnabled,
+            isEndpointRbacEnabled || isEndpointRbacV1Enabled,
```

Comment:
is there a use case for this? I purposely didn't want to over engineer it since we have pretty known use cases. since we added FF for the sake of privileges, it feels a bit odd to account for using the FF without privileges. this approach (in conjunction with the changes in `authz.ts`) controls privileges through the availability of subfeatures. since `isEndpointRbacV1Enabled` doesn't contain trusted apps subfeature, for example, trusted apps privilege would be false if only `isEndpointRbacV1Enabled` is enabled.

Reply:
I think from the change [here](https://github.com/elastic/kibana/pull/143991/files#diff-4e832c7ac3c1401b8a3238cfbd997baf3ad22804ee0612844398e04da66541b0R34) it will be evaluated if one of both is enabled. So for a user having `isEndpointRbacV1Enabled` set to true, we will evaluate privileges for `Trusted Apps`, `Event Filters` or others. And we should return false as it is not supported.

Reply:
Spoke offline, the control is handled by the change made in `features.ts`. So we are ok with this! Thanks for the clarification @joeypoon